esx_service/utils/auth_data.py

```diff
@@ -336,6 +366,7 @@ def connect(self):
         
         if need_create_table:
             self.create_tables()
+            self.create_fallback_tenant_and_privileges()
```

Comment:
This doesn't seem to be the right place to create fallbacks. I think we should add a flag to check if need_crreate_fallback. This way, we can handle the "upgrade" more smoothly. Also in the unit test we shouldn't have the logic to create fallbacks.